1001-2000/LIT1259Daniel.xml

```diff
@@ -49,7 +51,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
-         <ab/>
+         <div type="bibliography">
+            <listRelation>
+               <relation name="saws:formsPartOf" active="LIT1259Daniel" passive="LIT3529Daniel"/>
```

Comment:
this is correct if it is ALWAYS part of.
if it may have circulation without the other texts then you need the ecrm relation